### PR TITLE
Loosen MIN_IM_VERSION to 6.7.7

### PR DESCRIPTION
**Pull request by Watson1978**
Fix https://github.com/rmagick/rmagick/issues/474

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ os:
 sudo: required
 
 env:
+  - IMAGEMAGICK_VERSION=6.7.7-10
+  - IMAGEMAGICK_VERSION=6.7.7-10 CONFIGURE_OPTIONS=--enable-hdri
```
Comment (mockdeep) [inline .travis.yml]: Can we add this to `.appveyor.yml`, too?

Comment (Watson1978) [reply .travis.yml]: make sense. it sounds good👍

Comment (Watson1978) [reply .travis.yml]: After https://github.com/rmagick/rmagick/pull/340, we will add appveyor configuration.

**Comment by mockdeep** [comment]
@Watson1978 this lgtm. Is there anything else you want to do on it before we merge?

**Comment by Watson1978** [comment]
Ah, sorry. I just wait until it was merged.

**Comment by mockdeep** [comment]
Okie dokie, good to go!

**Pull request merged**